src/Scripting/VM/VirtualMachine.cpp

```
@@ -0,0 +1,1281 @@
+			case OVR_GL:
+			{
+				size_t consumed = 0;
+				AsTL::IDX idx = std::bit_cast<AsTL::IDX>(ins.operand);
+				m_globReg[idx] = castFromStack<AsTL::StackValue>(
+					static_cast<Compiler::OperandType>(
+						Compiler::StackValueToByte(m_globReg[idx])
+					),
+					&consumed
+				);
+
+				if (!insReadOnly())
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)